PyPi_Setup.md

```diff
@@ -0,0 +1,190 @@
+# PyPi setup 
+
+## 1. Introduction
+
+This documentation provides a comprehensive guide on how to set up and deploy a Python package to the Python Package Index (PyPI). The package is designed to facilitate the sharing of Python projects, enabling users to install it easily using pip.
```

Comment:
I feel like this can be better, it doesn't sound that good with that `This documentation` it should be something in the first person I feel. CC @this-is-yaash